src/android/src/main/java/org/pgsqlite/SQLitePlugin.java

```diff
@@ -76,62 +76,16 @@
     /**
      * Linked activity
      */
-    protected Activity activity = null;
+    protected Context context = null;
 
     /**
      * Thread pool for database operations
      */
     protected ExecutorService threadPool;
 
-    @Override
-    public void onActivityCreated(Activity activity, Bundle savedInstanceState) {
-
-    }
-
-    @Override
-    public void onActivityStarted(Activity activity) {
-
-    }
-
-    @Override
-    public void onActivityResumed(Activity activity) {
-
-    }
-
-    @Override
-    public void onActivityPaused(Activity activity) {
-
-    }
-
-    @Override
-    public void onActivityStopped(Activity activity) {
-
-    }
-
-    @Override
-    public void onActivitySaveInstanceState(Activity activity, Bundle outState) {
-
-    }
-
-    /**
-     * If activity matche linked Activity of this plugin, all open databases are closed.
-     *
-     * @param activity
-     */
-    @Override
-    public void onActivityDestroyed(Activity activity) {
-        Activity myActivity = this.getActivity();
-        if (activity == myActivity){
-            myActivity.getApplication().unregisterActivityLifecycleCallbacks(this);
-            Log.d(LOG_TAG, "linked activity destroyed - closing all open databases");
-            this.closeAllOpenDatabases();
-        }
-    }
-
-    public SQLitePlugin(ReactApplicationContext reactContext, Activity activity) {
+    public SQLitePlugin(ReactApplicationContext reactContext) {
```

Comment:
Seems like the SQLitePluginPackage class from pgsqlite is not included in this PR? Should be change in similar fashion as the liteglue class.